server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/bulkImport.js

```diff
@@ -16,99 +16,86 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+/* JSLint global definitions */
+/*global
+    $, document, sessionStorage, clearTableBody, EMPTY_ROW_THREE_CELLS, EMPTY_CELL, getBulkImports
+*/
 "use strict";
 
-/**
- * Creates bulk import initial table
- */
-$(document).ready(function() {
-  refreshBulkImport();
-});
-
-/**
- * Makes the REST calls, generates the tables with the new information
- */
-function refreshBulkImport() {
-  getBulkImports().then(function() {
-    refreshBulkImportTable();
-    refreshServerBulkTable();
-  });
-}
-
-/**
- * Used to redraw the page
- */
-function refresh() {
-  refreshBulkImport();
-}
-
 /**
  * Generates the manager bulk import status table
  */
 function refreshBulkImportTable() {
+    $("#bulkListTable tbody").html(EMPTY_ROW_THREE_CELLS);
 
-  clearTableBody('managerBulkImportStatus');
-
-  /*
-   * Get the bulk import value obtained earlier, if it doesn't exists,
-   * create an empty array
-   */
-  var data = sessionStorage.bulkImports === undefined ?
-      [] : JSON.parse(sessionStorage.bulkImports);
+    // Get the bulk import data from the session
+    var data = sessionStorage.bulkImports === undefined ?
+                [] : JSON.parse(sessionStorage.bulkImports);
 
-  /* If the data is empty, create an empty row, otherwise,
-   * create the rows for the table
-   */
-  if (data.length === 0 || data.bulkImport.length === 0) {
-    $('<tr/>', {
-      html: createEmptyRow(3, 'Empty')
-    }).appendTo('#managerBulkImportStatus tbody');
-  } else {
-    $.each(data.bulkImport, function(key, val) {
-      var items = [];
-      items.push(createFirstCell(val.filename, val.filename));
-      items.push(createRightCell(val.age, new Date(val.age)));
-      items.push(createRightCell(val.state, val.state));
-      $('<tr/>', {
-        html: items.join('')
-      }).appendTo('#managerBulkImportStatus tbody');
+    // If the data is empty, clear table, otherwise populate
+    if (data.length === 0 || data.bulkImport.length === 0) {
+        return;
+    }
+    console.log("Populate bulkListTable with " + sessionStorage.bulkImports);
+    var tableBodyHtml = "";
+    $.each(data.bulkImport, function (key, val) {
+        console.log("Append row " + key + " " + JSON.stringify(val) + " to bulkListTable");
+        tableBodyHtml += "<tr><td class='firstcell'>" + val.filename + "</td>";
+        tableBodyHtml += "<td class='center'>" + new Date(val.age) + "</td>";
+        tableBodyHtml += "<td class='center'>" + val.state + "</td></tr>";
     });
-  }
 
+    $("#bulkListTable tbody").html(tableBodyHtml);
 }
 
 /**
- * Generates the bulk import status table
+ * Generates the bulkPerServerTable table
  */
 function refreshServerBulkTable() {
+    // get the bulkImport data from sessionStorage
```

Comment:
```suggestion
function refreshServerBulkTable() {
    $("#bulkPerServerTable tbody").html(EMPTY_ROW_THREE_CELLS);

    // get the bulkImport data from sessionStorage
```
I tested out this branch and wasnt seeing the PerServer table updating when rows should have been removed. I think adding this will fix that.

Reply:
Yup! Thanks.